pkg/storj/node.go

```diff
@@ -184,3 +230,16 @@ func (n NodeIDList) Swap(i, j int) { n[i], n[j] = n[j], n[i] }
 
 // Less implements sort.Interface.Less()
 func (n NodeIDList) Less(i, j int) bool { return n[i].Less(n[j]) }
+
+func (id NodeID) versionByte() byte {
+	return id[NodeIDSize-1]
```

Comment:
is it possible NodeIDSize <= 0 ? or larger than len(id)+1 ?

Reply:
the length of `id` is `NodeIDSize`